### PR TITLE
fix: remove nullables from handlers

### PR DESCRIPTION
**Pull request by samucorea**
# Description

-Removed nullables to default values on several queries and commands
-Removed LoginResponse since we don't use it nor need it.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by gasimon12 — CHANGES_REQUESTED** [review]

File: NoNicotin_Business/Handler/CreatePatientCommandHandler.cs [hunk 1]
```diff
@@ -22,15 +22,13 @@ public class CreatePatientCommandHandler : IRequestHandler<CreatePatientCommand,
...
-        private readonly RoleManager<IdentityRole> _roleManager;
         private readonly ILogger<CreatePatientCommandHandler> _logger;
         private const string PATIENT_ROLE = "patient";
-        public CreatePatientCommandHandler(AppDbContext context, UserManager<IdentityUser> userManager, ILogger<CreatePatientCommandHandler> logger, RoleManager<IdentityRole> roleManager)
+        public CreatePatientCommandHandler(AppDbContext context, UserManager<IdentityUser> userManager, ILogger<CreatePatientCommandHandler> logger)
         {
             _context = context;
             _userManager = userManager;
             _logger = logger;
-            _roleManager = roleManager;
```
[the remark this answers is not in the document]
Comment (samucorea) [reply NoNicotin_Business/Handler/CreatePatientCommandHandler.cs:33]: yes

File: NoNicotin_Business/Handler/CreatePatientCommandHandler.cs [hunk 2]
```diff
@@ -144,6 +142,51 @@ public async Task<Response<Patient>> Handle(CreatePatientCommand request, Cancel
...
+            if (request.Sex == ' ')
+            {
+                return new Response<Patient>
+                {
+                    Message = "You must specify the patient sex",
+                    Succeeded = false
+                };
+            }
+
+            if(request.BirthDate == DateTime.Now)
```
[the remark this answers is not in the document]
Comment (samucorea) [reply NoNicotin_Business/Handler/CreatePatientCommandHandler.cs:154]: true, changing

File: NoNicotin_Business/Commands/CreatePatientCommand.cs [hunk 3]
```diff
@@ -14,15 +14,15 @@ namespace NoNicotin_Business.Commands
...
+        public string Name { get; set; } = string.Empty;
+        public char Sex { get; set; } = ' ';
+        public DateTime BirthDate { get; set; } = DateTime.Now;
+        public string Email { get; set; } = string.Empty;
+        public string Password { get; set; } = string.Empty;
 
-        public string? Identification { get; set; }
+        public string Identification { get; set; } = string.Empty;
 
-        public string IdentificationPatientType { get; set; } = IdentificationType.IDENTIFICATION_CARD;
```
[the remark this answers is not in the document]
Comment (samucorea) [reply NoNicotin_Business/Commands/CreatePatientCommand.cs:25]: I changed it to empty to force the client to specify an identification patient type.

[the remark this answers is not in the document]
Comment (gasimon12) [reply NoNicotin_Business/Commands/CreatePatientCommand.cs:25]: Understood, I'll do the same for the CreateTherapistCommand.

**Review by gasimon12 — APPROVED** [review]

**Pull request merged**